app/controllers/reports_controller.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+
+class ReportsController < BaseTraineeController
+
+  def index
+    @current_user = current_user
+  end
+
+  def itt_new_starter_data_signoff
+    respond_to do |format|
+      format.html
+      format.csv do
+        authorize(:trainee, :export?)
+        send_data(data_for_export.data, filename: "Itt-new-starter-data-signoff-2022.csv", disposition: :attachment)
+      end
+    end
+  end
+
+  private
+  def data_for_export
+    @data_export ||= Exports::TraineeSearchData.new(policy_scope(filtered_trainees))
+  end
+
+  def filtered_trainees
+    #will need to be made adaptive to year
+    census_date = date_of_nth_weekday(10, 2022 ,3 ,2)
```

Comment:
I don't think the year should be hardcoded like this. We should be using the app's current academic year.